x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/index.tsx

```diff
@@ -27,12 +26,8 @@ interface WizardState {
     | 'service';
   uploadType?: 'log-file' | 'api-key';
   elasticAgentPlatform: 'linux-tar' | 'macos' | 'windows' | 'deb' | 'rpm';
-  alternativeShippers: {
-    filebeat: boolean;
-    fluentbit: boolean;
-    logstash: boolean;
-    fluentd: boolean;
-  };
+  apiKeyId: string;
```

Comment:
Do we need to have this as part of the state now that step 3 and 4 were combined?

Reply:
good point i'll look into removing it